source/rst/lss_examples.rst

```diff
@@ -147,16 +148,16 @@ class called ``Samuelson``
 Notice, we didn't give the class a value for :math:`\Sigma_0`. In this
 case, the class automatically assumes that :math:`\Sigma_0 = 0`, i.e.
 the first period's values of the state vector are given by
-:math:`\mu_0`.
+:math:`\mu_0`
 
 The first method we will highlight is ``simulate()``
 
 Below we simulate :math:`C_t`, :math:`I_t` and :math:`Y_t` for 150
-periods.
+periods
 
 There is evidence in the plots of the "investment accelerator".
 Investment is highest in the first twenty periods, while :math:`Y_t` is
-growing.
+growing
 
 .. code:: ipython3
```

Comment:
You will need to change all

`.. code:: ipython3` to `.. code-block:: python3`